app/src/main/java/ar/uba/fi/mercadolibre/controller/ChatMessageController.java

```diff
@@ -0,0 +1,17 @@
+package ar.uba.fi.mercadolibre.controller;
+
+import java.util.List;
+
+import ar.uba.fi.mercadolibre.model.ChatMessage;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+
+public interface ChatMessageController {
+    @GET("chat-message/")
+    Call<APIResponse<List<ChatMessage>>> list();
+
+    @POST("chat-message/")
```

Comment:
primera vez que veo guiones medios en una URI. just saying

Reply:
jajaja edité el mensaje justo despues para que diga api rest, ojo

Reply:
https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris

Reply:
igual mas alla de que esto es una boludez, twitter lo hace con _ asi que ahora pusheo el cambio en app server a master, y lo meto aca tmb